ui/decredmaterial/theme.go

```diff
@@ -269,7 +269,7 @@ func Fill(gtx layout.Context, col color.NRGBA) layout.Dimensions {
 	return fill(gtx, col)
 }
 
-// mulAlpha scales all color components by alpha/255.
+// mulAlpha scales all color components.go by alpha/255.
```

Comment:
Kindly revert this... the  comments is talking about the component not the file.